### PR TITLE
[Doc] added instruction on how to update LS to 7.12+ running on JDK15+

### PR DESCRIPTION
**Pull request by andsel**
**PREVIEW:** https://logstash_12614.docs-preview.app.elstc.co/guide/en/logstash/master/getting-started-with-logstash.html#ls-jvm

## What does this PR do?
Updates documentation of getting started explaining what to do when an existing LS installation is upgraded to 7.12+ and runs on JDK15+

## Why is it important/What is the impact to the user?

Describe how to change existing `log4j2.properties` and `jvm.options` to run with JDK15 after an update to Logstash `7.12+`

### CODE REVIEW

File: docs/static/jvm.asciidoc [hunk 1]
```diff
@@ -59,3 +59,63 @@ a tarball.
...
 installation, you may get an error message, and {ls} will not start properly.
+
+[float]
+[[jdk15-upgrade]]
+==== Using JDK 15
+
+{ls} supports JDK 15, but you need to update settings in `jvm.options` and
+`log4j2.properties` if: 
+
+* you are upgrading from  {ls} 7.10.x (or earlier) to 7.12 or later, and
```
Comment (andsel) [inline docs/static/jvm.asciidoc:70]:
```suggestion
* you are upgrading from  {ls} 7.11.x (or earlier) to 7.12 or later, and 
```

**Review by andsel — COMMENTED** [review]
@karenzone thanks for great rewording, I suggested only a fix for version number, LMKWYT

File: docs/static/jvm.asciidoc (hunk 1 above)
Comment (karenzone) [reply docs/static/jvm.asciidoc:70]: Yes!  That's an important change. Thank you.

Comment (andsel) [reply docs/static/jvm.asciidoc]: Fixed

**Review by karenzone — COMMENTED** [review]
Docs build cleanly and LGTM! Thanks for adding this important information.

**Pull request merged**